mon-pix/translations/en.json

```diff
@@ -125,6 +125,12 @@
         "label": "Search for an organization",
         "placeholder": "Select an organization",
         "searchLabel": "Keyword search"
+      },
+      "other-authentication-providers": {
+        "signup-heading": "Other means of registration",
+        "login-heading": "Other means of login",
```

Comment:
Pour être cohérent avec le commentaire précédent ça serait `Other methods of logins`